src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -203,56 +246,78 @@
 
 (rf/reg-event-fx :wallet/get-suggested-routes
  (fn [{:keys [db now]} [{:keys [amount]}]]
-   (let [wallet-address          (get-in db [:wallet :current-viewing-account-address])
-         token                   (get-in db [:wallet :ui :send :token])
-         transaction-type        (get-in db [:wallet :ui :send :tx-type])
-         collectible             (get-in db [:wallet :ui :send :collectible])
-         to-address              (get-in db [:wallet :ui :send :to-address])
-         receiver-networks       (get-in db [:wallet :ui :send :receiver-networks])
+   (let [wallet-address (get-in db [:wallet :current-viewing-account-address])
+         token (get-in db [:wallet :ui :send :token])
+         transaction-type (get-in db [:wallet :ui :send :tx-type])
+         collectible (get-in db [:wallet :ui :send :collectible])
+         to-address (get-in db [:wallet :ui :send :to-address])
+         receiver-networks (get-in db [:wallet :ui :send :receiver-networks])
          disabled-from-chain-ids (or (get-in db [:wallet :ui :send :disabled-from-chain-ids]) [])
-         test-networks-enabled?  (get-in db [:profile/profile :test-networks-enabled?])
-         networks                ((if test-networks-enabled? :test :prod)
-                                  (get-in db [:wallet :networks]))
-         network-chain-ids       (map :chain-id networks)
-         bridge-to-chain-id      (get-in db [:wallet :ui :send :bridge-to-chain-id])
-         token-decimal           (when token (:decimals token))
-         token-id                (if token
-                                   (:symbol token)
-                                   (str (get-in collectible [:id :contract-id :address])
-                                        ":"
-                                        (get-in collectible [:id :token-id])))
-         to-token-id             ""
-         network-preferences     (if token [] [(get-in collectible [:id :contract-id :chain-id])])
-         gas-rates               constants/gas-rate-medium
-         amount-in               (send-utils/amount-in-hex amount (if token token-decimal 0))
-         from-address            wallet-address
+         test-networks-enabled? (get-in db [:profile/profile :test-networks-enabled?])
+         networks ((if test-networks-enabled? :test :prod)
+                   (get-in db [:wallet :networks]))
+         network-chain-ids (map :chain-id networks)
+         bridge-to-chain-id (get-in db [:wallet :ui :send :bridge-to-chain-id])
+         token-decimal (when token (:decimals token))
+         token-id (if token
+                    (:symbol token)
+                    (str (get-in collectible [:id :contract-id :address])
+                         ":"
+                         (get-in collectible [:id :token-id])))
```

Comment:
This can be a utility:

```
(defn format-token-id
  [token collectible]
(if token
                    (:symbol token)
                    (str (get-in collectible [:id :contract-id :address])
                         ":"
                         (get-in collectible [:id :token-id]))))
```